spdx-utils/src/test/kotlin/SpdxDeclaredLicenseMappingTest.kt

```diff
@@ -50,5 +51,13 @@ class SpdxDeclaredLicenseMappingTest : WordSpec({
                 }
             }
         }
+
+        "be case-insensitve" {
+            val rubyLowerCase = SpdxDeclaredLicenseMapping.mapping["ruby's"]
+            val rubyUpperCase = SpdxDeclaredLicenseMapping.mapping["RUBY'S"]
+
+            rubyLowerCase shouldNotBe null
+            rubyLowerCase shouldBe rubyUpperCase
+        }
```

Comment:
To be dogmatic, this test only knows that the mapping has the type `Map<String, SpdxExpression>`, so it could have both keys, "ruby's" and "RUBY's". It is only valid because it knows that the mapping actually is a `SortedMap` using `String.CASE_INSENSITIVE_ORDER`. This test would give more confidence regarding the API (but I don't insist on it, same in the class below):

```kotlin
SpdxDeclaredLicenseMapping.mapping.forEach { (key, license) ->
  SpdxDeclaredLicenseMapping[key.toLowerCase()] shouldBe license
  SpdxDeclaredLicenseMapping[key.toUpperCase()] shouldBe license
  SpdxDeclaredLicenseMapping[key.toLowerCase().capitalize()] shouldBe license
}
```

Reply:
Done.